### PR TITLE
Removes the old mining vendor, NT marine vendor, NT marine port from TG, gives the raven a sec tech lite

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

So like

This removes all instances of the old mining vendor
Replaces with the new one

image

Adds a sectech lite the raven

image
*image is not of the raven*

Ports https://github.com/tgstation/tgstation/pull/60554
https://github.com/tgstation/tgstation/pull/60012
and https://github.com/tgstation/tgstation/pull/62714

All mentions of nanotrasen from this pr were removed, to make it so indies could wear this

image
image

Adds a NT marine vendor, unused for now. I had a vague idea on what admin ship to put them on, but thats for a later PR
sprites are from TGMC as usual
image

## Why It's Good For The Game

cotment

## Changelog
:cl:
refactor: how marine vendors are organized
add: Adds a new NT marine vendor
add: Adds a indie marine vendor
add: New NT marine armor, kept indie in case indies want to use it
add: Deployable component
del: Removes vector from the sectech
del: Old mining vendor
tweak: Replaces all instances of the old mining vendor with the new one
/:cl:

### CODE REVIEW

**Comment by Apogee-dev** [comment]
I'm actually not thrilled about the idea of giving every secoff ship a sectech lite. Not that they're an incredible bounty of equipment or anything, but some ships do intentionally limit the equipment available to their secoffs, and sectechs (via vouchers) are a source of guns like E-TARs and Vectors that can be shockingly potent, far more than the default weapons many ships provide to their secoffs, which are typically basic handguns like Commanders, .38 revolvers, and M1911s with the odd pump action shotgun here or there. That and the fancy energy guns aren't even thematically appropriate for most ships. That's Nanotrasen kit, there, and it really ought to only be provided to nanotrasen ships. NT has a pretty bad lack of defining identity as it is, making their gear even more widely available probably won't help.

short version: I don't want the secoffs on ships like the Riggs running around with E-TARs.

**Comment by rye-rice** [comment]
Removed sectech lite to shiptests with sec officer after a discussion in the discord,

**Comment by Kurgis** [comment]
This unused object path should be deleted: `/obj/item/vending_refill/mining_equipment_golem`, additionally golem vendors are missing a circuitboard which they previously had, which causes issues if deconstructed.

**Comment by rye-rice** [comment]
Oh, didn't think about that, thanks for the catch

**Comment by rye-rice** [comment]
above should have been adressed, again thank you for catching it

**Comment by rye-rice** [comment]
fucking

**Comment by triplezeta** [comment]
stalebot stalebot go away, come again in 14 days
